### PR TITLE
feat: add FluxRecord.getRow() with response data stored in List

### PR DESCRIPTION
**Pull request by michaelahojna**
Closes #

## Proposed Changes

Adding possibility of accessing response data in List FluxRecord.getRow().

In case of using pivot on data, where field contains labels that occur by default in the annotated CSV (f.e. "result" or "table"), could be duplicated column names in response. In that case FluxRecord.getValues() (Map), which can hold only unique keys, doesn't show complete data. This edge case is solved by using FluxRecord.getRow() (List).

## Checklist

- [ ] CHANGELOG.md updated
- [ ] Rebased/mergeable
- [ ] A test has been added if appropriate
- [ ] `mvn test` completes successfully
- [ ] Commit messages are [conventional](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] Sign [CLA](https://www.influxdata.com/legal/cla/) (if not already signed)

### CODE REVIEW

File: client-core/src/main/java/com/influxdb/query/FluxRecord.java [hunk 1]
```diff
@@ -50,6 +51,8 @@ public final class FluxRecord implements Serializable {
      */
     private LinkedHashMap<String, Object> values = new LinkedHashMap<>();
 
+    private ArrayList<Object> row = new ArrayList<>();
```
Comment (bednar) [inline client-core/src/main/java/com/influxdb/query/FluxRecord.java:54]: In Java world is preferred way to use generic interface and left side of assignment:

```suggestion
    private List<Object> row = new ArrayList<>();
```

File: client-core/src/main/java/com/influxdb/query/FluxRecord.java [hunk 2]
```diff
@@ -121,6 +124,14 @@ public Map<String, Object> getValues() {
         return values;
     }
 
+    /**
+     * @return record's columns
+     */
+    @Nonnull
+    public ArrayList<Object> getRow() {
```
Comment (bednar) [inline client-core/src/main/java/com/influxdb/query/FluxRecord.java:131]: Please use `generic` interface:

```suggestion
    public List<Object> getRow() {
```

File: examples/src/main/java/example/RecordRowExample.java [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+        System.out.println("---------------------------------- FluxRecord.getRow() ----------------------------------");
+        for (FluxTable fluxTable : tables) {
+            List<FluxRecord> records = fluxTable.getRecords();
+            for (FluxRecord fluxRecord : records) {
+                System.out.println(fluxRecord.getRow());
+            }
+        }
+
+        influxDBClient.close();
```
Comment (bednar) [inline examples/src/main/java/example/RecordRowExample.java:81]: Please use `try-with-resources` statement with auto-close capabilities: https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html

try(InfluxDBClient client = InfluxDBClientFactory.create("http://localhost:9999", token, org, bucket) {

}

**Review by bednar — CHANGES_REQUESTED** [review]
Thanks for your PR 👍 
There are a few requirements that must be be satisfy before we accept the PR:

**Review by bednar — APPROVED** [review]
LGTM 🌹
